docs/engineering/front-end-development.md

```diff
@@ -5,66 +5,65 @@
 * [CSS/Sass](#css-sass)
 * [Javascript](#javascript)
 * [Drupal](#drupal)
-  		\- [Javascript in Drupal](#javascript-drupal)
+  * [Javascript in Drupal](#javascript-drupal)
 * [Accessibility](#accessibility)
 * [Images](#images)
 * [Relationship with UX/Design](#ux-design)
 
 ## <a name="css-sass"></a>CSS/Sass
 
 * Our CSS should be:
-  		\- **Predictable**
-  		\- **Reusable**
-  		\- **Maintainable**
-  		\- **Scalable**
+  * **Predictable**
+  * **Reusable**
+  * **Maintainable**
+  * **Scalable**
 * We practice responsive design starting mobile first, so our attention and practice is placed on the constrained environment and progressively enhanced.
 * We use Sass to preprocess our CSS.
 * We organize our Sass partials in a logical file structure convention that follows [Drupal 8 file organization practices](https://www.drupal.org/node/1887922):
-  		\- **Base** — CSS reset/normalize plus HTML element styling.
-  		\- **Layout** — macro arrangement of a web page, including any grid systems.
-  		\- **Component** — discrete, reusable UI elements.
-  		\- **State** — styles that deal with client-side changes to components.
-  		\- **Theme** — purely visual styling (“look-and-feel”) for a component.
-  		\- Example:
-
-````css:
-             base: css/base/normalize.css css/base/elements.css
-             layout: css/layout/layout.css css/layout/layout--medium.css css/layout/layout--wide.css
-             component: css/components/button.css css/components/dropdown.css css/components/pagination.css css/components/tabs.css …
-             theme: css/theme/theme--light.css css/theme/theme--dark.css ```
-
-            * We write modular CSS that supports design components.
-            * Classes should be named to reflect the intent and purpose of the design element they represent. Eg. instead of using `button--red`, we can ask "Why is the button red?" and name it `notification__button` .
-````
-
-/* Component Rules */
+  * **Base** — CSS reset/normalize plus HTML element styling.
+  * **Layout** — macro arrangement of a web page, including any grid systems.
+  * **Component** — discrete, reusable UI elements.
+  * **State** — styles that deal with client-side changes to components.
+  * **Theme** — purely visual styling ("look-and-feel") for a component.
+  * Example:
+
+
+```
+base: css/base/normalize.css css/base/elements.css
+layout: css/layout/layout.css css/layout/layout--medium.css css/layout/layout--wide.css
+component: css/components/button.css css/components/dropdown.css css/components/pagination.css css/components/tabs.css ...
+theme: css/theme/theme--light.css css/theme/theme--dark.css
+```
+
+* We write modular CSS that supports design components.
+* Classes should be named to reflect the intent and purpose of the design element they represent. Eg. instead of using `button--red`, we can ask "Why is the button red?" and name it `notification__button`.
+
+/\* Component Rules */
```

Comment:
There's a bunch of broken code snippet markdown, but that predates remark. I made a trello card for it. https://trello.com/c/aauP47ei/119-fix-frontend-code-snippets